Assignment4/angelux101-a180068/task06.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1-_vR2IwNm2x7239j2AtP0S8khRUEwKhR
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This will not run in Python, you need to comment out this line. The same applies for other files